pytest.ini

```diff
@@ -21,3 +21,4 @@ filterwarnings =
     ignore:^Using the slower implmentation::cupy
     ignore:Using the slower implementation::cupy
     ignore:Jitify is performing a one-time only warm-up::cupy
+    ignore:.*scipy.misc.*:DeprecationWarning
```

Comment:
If you still needed to suppress `scipy.misc` `DeprecationWarning`s below, is this doing anything for us?

Reply:
I removed this. I was having problems with the DeprecationWarnings appearing as errors during the testing and it was pretty hard to get rid of them.

Reply:
Ok, it's actually needed during the `smoke-doctests`, otherwise errors are raised.

Reply:
https://app.circleci.com/pipelines/github/scipy/scipy/31583/workflows/882ad70b-30b4-4540-86af-49c151491be2/jobs/106815